homeassistant/const.py

```diff
@@ -1040,7 +1040,9 @@ class UnitOfVolumeFlowRate(StrEnum):
     """Volume flow rate units."""
 
     CUBIC_METERS_PER_HOUR = "m³/h"
-    CUBIC_FEET_PER_MINUTE = "ft³/m"
+    CUBIC_FEET_PER_MINUTE = "ft³/min"
```

Comment:
Just a thought... do we not need to add this to "equivalent units"?
https://github.com/home-assistant/core/blob/bc720b48b476d2fa1ad754166d6aca72e0d180cf/homeassistant/components/sensor/recorder.py#L49-L55

Reply:
Not forgotten, just no timeslot yet. Yes i think so.

Reply:
https://github.com/home-assistant/core/pull/109501